File: user-service-java/src/main/java/com/project/user_service/services/UserService.java
package com.project.user_service.services;

import com.project.user_service.models.Role;
import com.project.user_service.models.User;

import java.util.List;

public interface UserService {
    User getUserById(long id);

    List<User> getAllUsers();

    void updateUserRole(long userId, Role role);
}

File: telegram-bot/user_service/user_service.py
class UserService:
    def get_user_by_id(self, id: int):
        raise NotImplementedError

    def get_student_by_user_id(self, user_id: int):
        raise NotImplementedError

    def is_student(self, user_id: int) -> bool:
        raise NotImplementedError
